Chat App/Commons/Constants.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 import UIKit
 
 struct Constants {
```

Comment:
Maybe create another level per origin. Something like

```
struct Constants {
struct Colors {
}
struct Strings {
}
}
``` For this it's better to use enum also

Reply:
Added enums to hold related constants by origin. Refactored references to these constants. These changes will be included in the next commit.